content/topics/templating.md

```diff
@@ -12,17 +12,10 @@ packages = [
   "askama",
   "tera",
   "handlebars",
-  "handlebars-iron",
-  "sailfish",
-  "horrorshow",
   "liquid",
-  "yarte",
   "maud",
   "markup",
   "pulldown-cmark",
-  "ramhorns",
-  "ructe",
-  "axohtml",
```

Comment:
I assume this one got removed because of low download numbers?

Reply:
Absolutely ☺️